packages/core/src/Input/Input.js

```diff
@@ -196,6 +202,10 @@ Input.defaultProps = {
  * @prop {string} [value]
  * @prop {string} [tabIndex]
  *
+ * @prop {string} [max] The native `max` attribute
+ * @prop {string} [min] The native `min` attribute
+ * @prop {string} [step] The native `step` attribute
```

Comment:
I see that here we're specifying the type as a string, but the accepted prop-type is a number. I took a look, and it seems that react will convert numbers passed to native html elements to strings (which is what native html elements expect). So both will work, but we should probably follow a single convention (string, number, or string || number).

I'm not sure what our convention is though.

Reply:
I talked to Jan-Gerke and he mentioned that we don't have a convention for this yet. Seems like we could just start conservatively and only accepts strings. And then if there's a reason for accepting numbers as well we can always accept that later on.

Reply:
Woops! this was a mistake - I meant to make them all `number` after copying and pasting from definitions above. Thanks for catching it 😉

Reply:
Oh weird - your second comment there wasn't visible when I posted my comment above, but its timestamp is before 🤔 

I will switch the prop types to strings then

Reply:
I agree with @ismay and @Mohammer5  that we should (for now) only accept strings. As @ismay says, React will accept numbers but will cast them to strings because that's what DOM element attributes are. That means that if we accept a number on a prop we use to control a DOM element's attribute, and later have to read the value of the value of that attribute, we get a type mismatch. For specifying `min`, `max` and `step` in particular, I don't really expect any problems. However, for something like `value` it can be problematic, for example:
```jsx
const NumberInput = (value, onChange) => <input type="number" value={value} onChange={onChange} />
NumberInput.propTypes = {
    value: PropTypes.number,
    onChange: PropTypes.func,
}
const UncontrolledNumberInput = () => {
    const [value, setValue] = useState(1)
    return <NumberInput value={value} onChange={event => setValue(event.target.value)} />}
}
```
In the example above `value` will first be a `number` with value `2`. However, once the users changes that value, `event.target.value` will be of type `string` and so will the state. We'll now also see a props error on the `UncontrolledNumberInput`. We could of course fix this by relaxing the prop-types a bit, but I really don't like props changing value type across render cycles.

The above example is probably pretty specific to `value`, but since `value` is a DOM attribute, just like `min`, `max`, and `step`, I think we should treat them all the same and only accepting a `string` for DOM attributes seems like the best solution to avoid ambiguity.

Reply:
The reason why I argued for strings only, even for number-type `value` values (:sweat_smile:) is:
Our current strategy is to us RFF, which provides the parse/format functionality, so it's something that's not really an issue to dhis2 right now.

3rd party dhis2 apps and other apps would face the same problem, regardless of the prop type definition anyways, as the dom element converts the value to a string anyway, so they'll have to figure out a solution regardless of what we do.

Reply:
> The above example is probably pretty specific to value, but since value is a DOM attribute, just like min, max, and step, I think we should treat them all the same and only accepting a string for DOM attributes seems like the best solution to avoid ambiguity.

That sounds like a good reason 👍. Plus we can always add to it later, whereas removing an accepted proptype will be more difficult.

Reply:
Strings it is 👍